### PR TITLE
[EIP1-3490] Hotfix build

### PR DESCRIPTION
**Pull request by stalwartsolutions**
- Normal releases now use minor versions (v1.X.0)
- Hotfixes now use patch versions (v1.1.X)
- Changes to hotfix/v* branches (after the initial push) are tagged and deployed to Test.

### CODE REVIEW

**Review by NathanRussellValtech — APPROVED** [review]
I understand the new process 👍 , but cant say I fully understand/grok the actual implementation here, but I trust ya! 😁 

LGTM I guess ¯\_(ツ)_/¯

File: .github/workflows/build-and-push.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        with:
+          aws-access-key-id: ${{ secrets.LIVE_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.LIVE_AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-2
+
+      - name: Login to Amazon ECR (live env)
+        id: login-ecr-live
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build, tag, and push image to Amazon ECR (live env)
```
Comment (Neil-Massey) [inline .github/workflows/build-and-push.yml:65]: Doesn't look like it actually builds it - more Promote?

File: .github/workflows/hotfix.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          repository: cabinetoffice/eip-ero-print-api
+          releases-only: false
+          prefix: '${{ env.BRANCH_VERSION }}'
+
+      - name: Bump patch version tag
+        uses: cabinetoffice/action-bump-semver@v1
+        id: bump-semver
+        with:
+          current_version: ${{ steps.get-latest-tag.outputs.tag }}
+          level: patch
```
Comment (Neil-Massey) [inline .github/workflows/hotfix.yml:56]: 👍

File: .github/workflows/build-and-push.yml (hunk 1 above)
Comment (stalwartsolutions) [reply .github/workflows/build-and-push.yml:65]: Fair

File: .github/workflows/build-image-and-deploy-to-dev.yml [hunk 3]
```diff
@@ -44,54 +26,27 @@ jobs:
         id: bump-semver
         with:
           current_version: ${{ steps.get-latest-tag.outputs.tag }}
-          level: patch
+          level: minor
```
Comment (Neil-Massey) [inline .github/workflows/build-image-and-deploy-to-dev.yml:29]: 👍

**Review by Neil-Massey — APPROVED** [review]
LGTM - Minor observation

**Pull request merged**